Quartex.TableStorage.Benchmark/Benchmarks/GetDocumentsBenchmark.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Azure.Data.Tables;
+using BenchmarkDotNet.Attributes;
+using Quartex.TableStorage.Benchmark.Data;
+using Quartex.TableStorage.Benchmark.Models;
+using Quartex.TableStorage.Benchmark.Storages;
+
+namespace Quartex.TableStorage.Benchmark.Benchmarks;
+[MemoryDiagnoser]
+public class GetDocumentsBenchmark
+{
+    private readonly List<DocumentDto> _documents = DataGenerator.GenerateDocuments(10000, true, 20);
+    private readonly TableClient _documentsTableClient = TableClientFactory.DocumentClient();
+    private readonly TableClient _metatadaTableClient = TableClientFactory.MetadataClient();
+    private readonly TableClient _slimDocumentsTableClient = TableClientFactory.SlimDocumentClient();
+
+    [GlobalSetup]
+    public void Setup()
+    {
+
+        _documentsTableClient.Delete();
+        _metatadaTableClient.Delete();
+        _slimDocumentsTableClient.Delete();
+
+        var insertTasks = new List<Task>();
+        var documentStorage = new DocumentStorage(_documentsTableClient);
+        insertTasks.Add(documentStorage.InsertPaginatedAsync(_documents, 100));
+
+        var storage = new SlimDocumentStorage(_slimDocumentsTableClient, new MetadataStorage(_metatadaTableClient));
+        insertTasks.Add(storage.InsertPaginatedAsync(_documents, 100));
+
+        Task.WhenAll(insertTasks).ConfigureAwait(false).GetAwaiter().GetResult();
+
+        _documentsTableClient.CreateIfNotExists();
+        _metatadaTableClient.CreateIfNotExists();
+        _slimDocumentsTableClient.CreateIfNotExists();
+    }
+
+
+    [Benchmark]
+    public async Task Get_100_On_Page_50_Documents_PagedWithoutPagingEntity()
```

Comment:
```suggestion
    public async Task Get100Documents_Page50_WithoutPagingEntity()
```
Would a naming convention like this work better?

I think it reads more like a sentence "Get 100 documents on page 50 without a paging entity".

The word "Paged" doesn't seem to add anything and the underscores don't seem to add much. Plus moving the word "Documents" seems to help with the sentence like structure.